src/vs/platform/audioCues/browser/audioCueService.ts

```diff
@@ -152,15 +165,15 @@ export class AudioCueService extends Disposable implements IAudioCueService {
 		Event.filter(this.configurationService.onDidChangeConfiguration, (e) =>
 			e.affectsConfiguration('audioCues.enabled')
 		),
-		() => /** @description config: audioCues.enabled */ this.configurationService.getValue<'on' | 'off' | 'auto'>('audioCues.enabled')
+		() => /** @description config: audioCues.enabled */ this.configurationService.getValue<'on' | 'off' | 'auto' | 'userGesture' | 'always' | 'never'>('audioCues.enabled')
 	);
 
-	private readonly isEnabledCache = new Cache((cue: AudioCue) => {
+	private readonly isCueEnabledCache = new Cache((cue: AudioCue, userGesture?: boolean) => {
```

Comment:
Unfortunately, the `Cache` doesn't work like this - you can only set one parameter.
I think multi parameter cache makes sense, so feel free to add this functionality to the Cache.

I just wonder why the compiler didn't complain here.